usr/src/lib/libfakekernel/common/sys/cmn_err.h

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Did you run into problems using the normal sys/cmn_err.h?
What happened?

Reply:
Some of the print functions have different prototypes (kernel versions are void, user space versions return number of characters printed) and this causes the compiler to generate an error.